github/src/main/java/com/novoda/github/reports/service/network/GithubApiService.java

```diff
@@ -71,6 +72,7 @@
             @Query("per_page") int perPageCount
     );
 
+    @Deprecated
```

Comment:
this has been deprecated for some time. everything `TimelineEvent` has. just added the annotation to make it obvious to anyone trying to use it in the future. but we should look into removing it.